project/src/main/java/com/google/autograder/servlets/auth/RequestTokenServlet.java

```diff
@@ -31,7 +31,8 @@
     private static String ACCESS_TYPE = "offline";
     private static String RESPONSE_TYPE = "code";
 
-    private static String SCOPE = "https://www.googleapis.com/auth/classroom.courses.readonly";
+    private static String SCOPE = "https://www.googleapis.com/auth/classroom.courses.readonly "
+                                + "";
```

Comment:
nit: ?

Reply:
Oh thanks for catching that, I was experimenting with multiple scopes in a single OAuth request.